components/Layout/Navbar.tsx

```diff
@@ -141,23 +141,26 @@ const Navbar: FC = () => {
         borderColor={$colors.border}
         justifyContent="space-between"
       >
-        <Link href={RoutePaths.Home} passHref>
-          <Box
-            order={1}
-            justifySelf="flex-start"
-            flex={{ base: null, sm: 1 }}
-            mr={{ base: 0, sm: '1.5rem' }}
-            w="50%"
-            mb="0.125rem"
-            whiteSpace="nowrap"
-          >
-            <IronFishLogo cursor="pointer" />
-            <NodeVersionButton
-              mx="1rem"
-              display={{ base: 'none', lg: 'inline-block' }}
-            />
-          </Box>
-        </Link>
+        <Flex
+          order={1}
+          alignItems="center"
+          justifySelf="flex-start"
+          flex={{ base: null, sm: 1 }}
+          mr={{ base: 0, sm: '1.5rem' }}
+          w="50%"
+          mb="0.125rem"
+          whiteSpace="nowrap"
+        >
+          <Link href={RoutePaths.Home} passHref>
+            <Flex as="a" alignSelf="stretch" alignItems="center">
+              <IronFishLogo cursor="pointer" />
+            </Flex>
+          </Link>
+          <NodeVersionButton
```

Comment:
Do you mind fixing up whatever this button uses to use anchor tags as well? I just happened to notice it when clicking around on the header a bit